src/main/java/com/e451/domain/Transaction.java

```diff
@@ -0,0 +1,140 @@
+package com.e451.domain;
+
+/**
+ * Created by e978726 on 2/24/2017.
+ */
+public class Transaction {
+    private Long id;
+    private Long householdKey;
+    private Long basketId;
+    private Long day;
+    private Long productId;
+    private Long quantity;
+    private Double salesValue;
+    private Long storeId;
+    private Double couponMatchDiscount;
+    private Double couponDisc;
+    private Double retailDisc;
+    private Long transTime;
+    private Long weekNo;
+
+    public Transaction(Long id, Long householdKey, Long basketId, Long day, Long productId, Long quantity, Double salesValue, Long storeId, Double couponMatchDiscount, Double couponDisc, Double retailDisc, Long transTime, Long weekNo) {
```

Comment:
This is a really long line of code. Maybe break it up into multiple lines to make it more readable:

```
    public Transaction(Long id, Long householdKey, Long basketId,
                      Long day, Long productId, Long quantity,
                      ...) {
    ...
    }
```